### ISSUE TITLE
OpenRASP is correctly installed but it does not block the test cases 

### ISSUE THREAD

**Issue by bbossola** [baidu/openrasp#156]
I installed OpenRASP on a fresh copy of Tomcat 8.5.6 and I also installed the tests (vuln.war). It looks like the setup was successful as:
- I can see the OpenRASP logo in the Tomcat log
- I can see the "X-Protected-By: OpenRASP" and "X-Request-ID: ..." headers in any curl request
However, the tests are not blocked, I can see for example 001 and 004 go through happily.

Steps:
- install a fresh copy of Tomcat
- deploy the vuln.war app 
- start Tomcat using catalina.sh, 
- see vulnerabilities 001 and 004 exploited
- stop tomcat
- install OpenRASP 
- start tomcat
- see the OpenRASP logo in the tomcat log
- see the "X-Protected-By: OpenRASP" and "X-Request-ID: ..." headers in any curl request
- see vulnerabilities 001 and 004 exploited <== NOT EXPECTED!!!
After a successful OpenRASP installation, I expect the vulnerability to be blocked and to see the blocking page.

Environment information:
- OS: Ubuntu 14.04.5 LTS \n \l
- Application server: Apache Tomcat 8.5.6 / 8.5.34 (tried both) 
- Java version: 1.8.0_171
- OpenRASP version: 0.42 (also tried 0.40)

Please advise.

**Comment by bbossola** [comment]
On a side note, with RASP 0.31 case 001, 002 and 004 are correctly blocked as expected. Maybe a regression in the last version?

**Comment by CaledoniaProject** [comment]
Bruno,  thanks for testing. Did you download the testcases from [baidu-security/openrasp-testcases](https://github.com/baidu-security/openrasp-testcases) ?

I've just tested them again (master branch), with the default payload provided on each HTML page:
1. case-001 is blocked
2. case-004-1 is blocked, regardless of the command in use
3. case-004-2 is not blocked, this is a known issue. We have a command injection detector that will be released in the upcoming 1.0.0 RC1 release, which should solve this issue.

So, for case-001, what payload are you using?

**Comment by CaledoniaProject** [comment]
Also, in version prior to v0.40, we blocked all command execution by default, that how case-004-2 gets blocked with OpenRASP agent v0.31.

I got plenty of user feedbacks, they need to execute commands like `nbtscan` or `python` on their production environments. So we turned this algorithm off by default.

We have detailed [customization document here](https://rasp.baidu.com/doc/dev/official.html), but it's currently in mandarin, we'll do a complete translation when version 1.0.0 final is out.
